pages/api/Using_with_nmap.md

```diff
@@ -29,14 +29,15 @@ This is the shape of the expected payload.
    "hosts": [
      {
        "ip_address" : "32.199.2.33",
-	      “ptr_record”: “abc-32-199-2-33.cloudfront.net”,
+	      "ptr_record": "abc-32-199-2-33.cloudfront.net",
```

Comment:
Redundant leading spaces. Should be inline with `"ip_address".`

Reply:
It was the "ip_address" line the one out of line/spaces. 👍